frontend/app/src/components/settings/OracleEntry.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <v-row align="center">
-    <v-col cols="auto">
+    <v-col cols="4">
       <v-img
         :width="size"
         contain
         position="left"
-        :max-height="size"
+        :max-height="height"
```

Comment:
```suggestion
        :height="size"
```
Height is not defined anywhere in the component though, right?

Also instead of changing the size for the manual entry how it would work if you changed the size property to 40 for all the elements, would the other icons still be smaller than the book one?

Reply:
Changing this it still looks bigger 

![image](https://user-images.githubusercontent.com/5068010/125590856-b379ef34-8003-4660-9b25-729e2402edba.png)